### DISCUSSION TITLE
Change the order of chart lines

### DISCUSSION THREAD

**Discussion by da-jacob** [apexcharts/apexcharts.js#3929] [Q&A] [answered]
So here's the thing. I got a chart with two datasets, Visitors and Orders. Is there any way to change the order in which are the lines displayed without changing the order of datasets in the tooltip?

On the image below, orange line represents visitors and purple line represents orders but in the tooltip the order is reversed, so are visitors on top and orders on bottom. I find that little bit confusing to read.

<img width="1290" alt="image" src="https://github.com/apexcharts/apexcharts.js/assets/42645271/1c0fdfab-f948-4be7-a749-cd70ea876d64">

**Comment by easyperl** [comment, marked as answer]
So this would reverse the tooltip when set to true... which I think would be fine to resolve confusion:
![image](https://github.com/apexcharts/apexcharts.js/assets/62156734/60871094-d1f0-4af9-a281-4831b201dfa3)

I am not sure how you would visibly reverse the lines... obviously it would depend on scale as I feel like this might be a dual y axis (opposite: true) chart with one series on each opposing Y.  If so, just change the order of the data/opposite.

If this is not, you can change the order of the series data and the colors would change... but you aren't showing the color/marker in the tooltip so I am not sure how it would help.  Your tooltip series will swap as well.

Maybe more information would help me/us understand if the above doesn't answer it, or possibly you already resolved it yourself?

**Reply by da-jacob** [reply]
inverseOrder is exactly what I was looking for. Thank you!
